docs/dev/openstack-howto.md

```diff
@@ -0,0 +1,93 @@
+# OpenStack HOWTO
+
+It's possible to launch an all-in-one - for test and development - cluster on OpenStack.
```

Comment:
This is fine, but the text would read better if we said something like 

"For test and development it's possible to deploy an all-in-one OpenStack environment, then deploy OpenShift on that cluster using openshift-install.  Below we provide an overview of a typical all-in-one development environment, including some scripts to automate the OpenStack install.  For OpenShift installation on existing OpenStack clouds, please refer to <user facing docs started by @russellb>"